src/Components/Cell/index.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import "./Cell.css";
+const Cell = ({ cellId, value, handleClick }) => {
+  return (
+    <>
+      <div
+        className={`individual-cell ${
+          value === "X" ? "blue-color-text" : "red-color-text"
+        }`}
```

Comment:
Use the [`classnames`](https://www.npmjs.com/package/classnames) package to give conditional class names.